checklists.md

```diff
@@ -0,0 +1,293 @@
+described in the article is this:
+https://apps.who.int/iris/bitstream/handle/10665/44186/9789241598590_eng_Checklist.pdf
+
+![who-surgery-checklist](https://user-images.githubusercontent.com/194400/56948566-68a4b780-6b28-11e9-8c36-4411928e61df.png)
+
+> Notice how at the bottom of the single-page checklist it says:
+"_This checklist is not intended to be comprehensive.
+**Additions and modifications** to fit local practice
+are **encouraged**._"
+The WHO does not give people the _tools_ to modify the checklist.
+See:
+https://www.who.int/patientsafety/topics/safe-surgery/checklist/en/
+What if it was a _single click_ to "***Customise This Checklist***"
+and the UX/UI for doing so was _flawless_ on any device ...?
+That is what we are going to achieve.
+
+
+The BBC article was shared on and reached the _top_ of HackerNews
+with over a 1k points
+(_that's a lot for such a short and relatively "boring" post_!)
+https://news.ycombinator.com/item?id=19682451
+![image](https://user-images.githubusercontent.com/194400/56955462-672fbb00-6b39-11e9-904e-86fdcc276d7a.png)
+
+The top comment has an insightful description of checklists:
+"_a way to **debug your life**_".
+
+Reading through the comments on HN,
+it's clear that _many_ engineers understand the
+benefits/power of a checklist in their work/lives,
+however there are still a few who remain to be convinced.
+
+
+<!--
+The top _negative_ review on Amazon shows how some people
+just don't "_get_" it. Anthony J. Vera
+
+![image](https://user-images.githubusercontent.com/194400/56947949-853ff000-6b26-11e9-94b5-927daa7c5184.png)
+
+The most _helpful_ review is:
+![image](https://user-images.githubusercontent.com/194400/56963336-9140a780-6b50-11e9-8a27-6f209e2a61fe.png)
+
+
+-->
+
+Atul's book "The Checklist Manifesto",
+is his attempt to make the benefits of checklists
+more widespread in more areas of life and society.
+
+
+A good checklist is:
++ Practical -
++ Concise - focus on the important parts of the task
++ Speedy - it should not add "overhead" to the task/activity.
++ 5–9 items (_though they can be longer when necessary,
+  see WHO checklist below_)
++ 1 page max -
++ A clear "Pause Point" where you stop during the completion
++ _Supplement_ existing knowledge and expertise not _replace_ it.
++ Uses Upper- & Lowercase
++ _Field-Tested_ in the real world, not theoretical.
+
+
+
+## Why do _we_ care about checklists?
+
+We care about checklists because
+they are the _single_ most effective way of achieving _any_ objective
+
+Anything from giving a housekeeper instructions
+for how to use the
+
+
+## "Old Dogs, New Tricks"
+
+One of the areas I'm _most_ excited about is learning/teaching!
+
+
+
+# _What?_
+
+Our mission is simple:
+build the simplest possible UI/UX
+for creating, sharing and performing repeatable checklists.
+
+And in doing this we will _systematically_:
+1. Build a tool that we will use ourselves to:
+  1. maximise our _personal_ effectiveness
+  2.
```

Comment:
I don't think the indentation is correct on this list

Reply:
@SimonLab good spot. I've added the missing bullet point.